docs/sources/datasources/tempo/traces-in-grafana/link-trace-id.md

```diff
@@ -69,10 +72,10 @@ You can link to Tempo traces from logs or metrics.
 
 You can link to Tempo traces from logs in Loki, Elasticsearch, Splunk, and other logs data sources by configuring an internal link.
 
-To configure this feature, see the [Derived fields]({{< relref "../loki#configure-derived-fields" >}}) section of the Loki data source docs or the [Data links]({{< relref "../elasticsearch#data-links" >}}) section of the Elasticsearch or Splunk data source docs.
+To configure this feature, refer to the [Derived fields]({{< relref "../../loki#configure-derived-fields" >}}) section of the Loki data source docs or the [Data links]({{< relref "../../elasticsearch#data-links" >}}) section of the Elasticsearch or Splunk data source docs.
```

Comment:
Let's ditch these relrefs since we're modifying the line already

Reply:
```suggestion
To configure this feature, refer to the [Derived fields](../../loki#configure-derived-fields) section of the Loki data source docs or the [Data links](../../elasticsearch#data-links) section of the Elasticsearch or Splunk data source docs.
```